index.html

```diff
@@ -48,37 +48,33 @@
 
   // editors, add as many as you like
   // only "name" is required
-  editors: [      {
-      name: "Manu Sporny",
-      url: "http://manu.sporny.org/",
-      company: "Digital Bazaar",
-      companyURL: "https://digitalbazaar.com/",
-      w3cid: 41758
-    },
-    {
-      name: "Markus Sabadello",
-      url: "https://www.linkedin.com/in/markus-sabadello-353a0821",
-      company: "Danube Tech",
-      companyURL: "https://danubetech.com/",
-      w3cid: 46729
-    }, {
-      name: "Drummond Reed",
-      url: "https://www.linkedin.com/in/drummondreed/",
-      company: "Evernym",
-      companyURL: "https://www.evernym.com/",
-      w3cid: 3096
-    }
-  ],
+  editors: [{
+    name: "Orie Steele",
+    url: "https://www.linkedin.com/in/or13b/",
+    company: "Transmute",
+    companyURL: "https://www.transmute.industries/"
```

Comment:
Hey @OR13, I don't know your `w3cid`. Please put in a change request under this line that looks like the one I have for my entry below. You can get your W3C ID by visiting: https://www.w3.org/users/myprofile ... log in, then the URL will show you your ID.

Reply:
I'm still learning github suggestions ui... I assume I'm not supposed to commit these suggestions in this batch.

Reply:
You could have committed the suggestions in a batch since I was asking you for the suggestions. I'll commit now and merge.